cmake/Modules/FindFLANN.cmake

```diff
@@ -19,41 +19,41 @@ endif(FLANN_USE_STATIC)
 
 find_package(PkgConfig QUIET)
 if (FLANN_FIND_VERSION)
-    pkg_check_modules(PC_FLANN flann>=${FLANN_FIND_VERSION})
+    pkg_check_modules(FLANN flann>=${FLANN_FIND_VERSION})
 else(FLANN_FIND_VERSION)
-    pkg_check_modules(PC_FLANN flann)
+    pkg_check_modules(FLANN flann)
 endif(FLANN_FIND_VERSION)
 
-set(FLANN_DEFINITIONS ${PC_FLANN_CFLAGS_OTHER})
```

Comment:
Is FLANN no longer exporting definitions?

Reply:
Ubuntu 16.04, `$ pkg-config --cflags-only-other flann` yields empty string.